plugins/module_utils/_api_data.py

```diff
@@ -2984,8 +2984,8 @@ def join_path(path):
                 'jump-target': KeyInfo(can_disable=True),
                 'layer7-protocol': KeyInfo(can_disable=True),
                 'limit': KeyInfo(can_disable=True),
-                'log': KeyInfo(can_disable=True),
-                'log-prefix': KeyInfo(can_disable=True),
+                'log': KeyInfo(default=False),
+                'log-prefix': KeyInfo(default=''),
```

Comment:
What's the reason behind this change? Was the behavior always wrong, or did RouterOS's behavior change from one version to another?

Reply:
<img width="143" alt="image" src="https://github.com/user-attachments/assets/082386cc-1af3-4a6d-8e62-f2f923310ae4">
As far as I'm concerned, the behavior was always wrong. I have tested on 6.49.10, 6.49.13, 7.14.1. If you use the --diff parameter in asible, you will see that each time ansible tries to remove these fields.